cmake/Templates/MaCh3Config.cmake.in

```diff
@@ -45,8 +45,16 @@ endif()
 if(MaCh3_GPU_ENABLED)
   enable_language(CUDA)
   add_compile_definitions(CUDA)
+  #KS: Maybe just include CUDASetup.cmake or make cmake more robust rather than copy paste?
   find_package(CUDAToolkit)
   set(CUDA_ARCHITECTURES @CUDA_ARCHITECTURES@)
+  if(NOT DEFINED ENV{CUDAPATH})
+    cmessage(FATAL_ERROR "CUDAPATH environment variable is not defined. Please set it to the root directory of your CUDA installation.")
+  endif()
+  include_directories(
+    $ENV{CUDAPATH}/include
+    $ENV{CUDAPATH}/samples/common/inc>
```

Comment:
I have seen these installed outside of the $CUDAPATH variable on one of the NERSC systems - Is there any way to check the target is correctly added to ensure it's found?